src/Components/AddBook.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function AddBook() {
+  return (
+    <div>
+      <form action="#" className="form">
+        <input type="text" />
+        <button type="button">Add</button>
+      </form>
+    </div>
```

Comment:
The inputs for the add book form should be title and author as per project requirements.

![image](https://user-images.githubusercontent.com/38088818/193290542-35cfd66c-bf0a-4210-93af-a79776232502.png)

- Kindly include the input field for the author and make sure it is displayed according to the design layout.

Reply:
Okay, thanks. I will add it.